src/lib/publicSuffixList.js

```diff
@@ -85,6 +85,7 @@ var publicSuffixes = {
     "adult.ht": 1,
     "adv.br": 1,
     "adv.mz": 1,
+    "advisor.ws": 2,
```

Comment:
PSL conversion (in `scripts/convertpsl.py`) and use (in `getBaseDomain`) is one of the holdover ABP bits. I treated it as a black box ... It seems the numeric value determines how many segments of the domain get removed when calculating base domain. Not sure though.